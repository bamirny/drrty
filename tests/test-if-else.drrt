
function int main(){

    if( False ){ 
        prints("Hello world"); 
    }
    else{ 
        prints("Sorry world");
    }
    
    return 0; 
}

