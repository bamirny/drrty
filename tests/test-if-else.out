<span>Sorry world</span><br>
